### ISSUE TITLE
error after upgrading to 1.2.0 with react native

### ISSUE THREAD

**Issue by sylvainbaronnet** [marceloadsj/react-indicative-hooks#1]
Hi,

I have this error after upgrading to 1.2.0

```
error: bundling failed: Error: While trying to resolve module `react-indicative-hooks` from file `/Users/admin/Documents/git/sparvest-mobile/sparvest/app/scenes/SignIn/index.js`, the package `/Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/react-indicative-hooks/package.json` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`/Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/react-indicative-hooks/dist/index.umd.js`. Indeed, none of these files exist:
  * `/Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/react-indicative-hooks/dist/index.umd.js(.native||.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)`
  * `/Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/react-indicative-hooks/dist/index.umd.js/index(.native||.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)`
    at ResolutionRequest.resolveDependency (/Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/metro/src/node-haste/DependencyGraph/ResolutionRequest.js:65:15)
    at DependencyGraph.resolveDependency (/Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/metro/src/node-haste/DependencyGraph.js:283:16)
    at Object.resolve (/Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/metro/src/lib/transformHelpers.js:261:42)
    at dependencies.map.result (/Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/metro/src/DeltaBundler/traverseDependencies.js:399:31)
    at Array.map (<anonymous>)
    at resolveDependencies (/Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/metro/src/DeltaBundler/traverseDependencies.js:396:18)
    at /Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/metro/src/DeltaBundler/traverseDependencies.js:269:33
    at Generator.next (<anonymous>)
    at asyncGeneratorStep (/Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/metro/src/DeltaBundler/traverseDependencies.js:87:24)
    at _next (/Users/admin/Documents/git/sparvest-mobile/sparvest/node_modules/metro/src/DeltaBundler/traverseDependencies.js:107:9)

```

**Comment by marceloadsj** [comment]
Hello @sylvainbaronnet!

Humm, I will take a look into this today and release a new version asap. :)

**Comment by marceloadsj** [comment]
@sylvainbaronnet I've fixed on the version 1.2.2.
Can you try and check if it works? :)

Tks and sorry for the delay!

**Comment by sylvainbaronnet** [comment]
yes it works 👍 Thanks for the quick fix !
